virt/virt-getting-started.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: ASSEMBLY
+[id="virt-getting-started"]
+include::_attributes/common-attributes.adoc[]
+= Getting started with {VirtProductName}
+:context: virt-getting-started
+
+toc::[]
+
+You can install and configure a basic {VirtProductName} environment in order to explore its features and functionality.
```

Comment:
Suggestion >>You can explore the features and functionalities of  {VirtProductName} by installing and configuring a basic environment.